### PR TITLE
operator cloud-native-postgresql (1.23.0)

### PR DESCRIPTION
**Pull request by cnp-autobot**
Automated changes by [create-pull-request](https://github.com/peter-evans/create-pull-request) GitHub action

### CODE REVIEW

File: operators/cloud-native-postgresql/1.23.0/manifests/cloud-native-postgresql.clusterserviceversion.yaml [hunk 1]
```diff
@@ -0,0 +1,1331 @@
...
+    createdAt: "2024-04-25T11:05:52Z"
+    description: EDB Postgres for Kubernetes is an open source operator designed to manage highly available PostgreSQL databases with a primary/standby architecture on any supported Kubernetes cluster.
+    features.operators.openshift.io/disconnected: "true"
+    features.operators.openshift.io/fips-compliant: "true"
+    features.operators.openshift.io/proxy-aware: "false"
+    features.operators.openshift.io/tls-profiles: "false"
+    features.operators.openshift.io/token-auth-aws: "false"
+    features.operators.openshift.io/token-auth-azure: "false"
+    features.operators.openshift.io/token-auth-gcp: "false"
+    olm.skipRange: '>= 1.18.0 < 1.23.0'
```
Comment (itsmitul9) [inline operators/cloud-native-postgresql/1.23.0/manifests/cloud-native-postgresql.clusterserviceversion.yaml:81]: skips is an array and must be formatted properly. (note GH isn't formatting the below properly for whatever reason)

skips:
   - cloud-native-postgresql.v1.22.0

Comment (jfrancin) [reply operators/cloud-native-postgresql/1.23.0/manifests/cloud-native-postgresql.clusterserviceversion.yaml:81]: @itsmitul9 wrap your code examples using the `code` Markdown wrapper - three backticks in a row.
```
skips:
  - cloud-native-postgresql.v1.22.0
```

Comment (litaocdl) [reply operators/cloud-native-postgresql/1.23.0/manifests/cloud-native-postgresql.clusterserviceversion.yaml:81]: @jfrancin @itsmitul9 

This error is regarding different ocp versions, this probably be a common issue if we want to support one more ocp versions, did the skips could ensure olm upgrade all versions in range '>= 1.18.0 < 1.23.0' ?

```
Replaces bundle Bundle(cloud-native-postgresql/1.22.0) ['4.12', '4.13', '4.14'] does not support the same OCP versions as bundle Bundle(cloud-native-postgresql/1.23.0) ['4.12', '4.13', '4.14', '4.15']
```

Comment (litaocdl) [reply operators/cloud-native-postgresql/1.23.0/manifests/cloud-native-postgresql.clusterserviceversion.yaml:81]: we have a similar case here https://github.com/redhat-openshift-ecosystem/certified-operators/pull/3797

The support help solved this pr weeks ago due to same error https://github.com/redhat-openshift-ecosystem/certified-operators/pull/3771.

**Pull request closed without merging**